examples/mobile/HomeApp/js/app.js

```diff
@@ -112,6 +101,57 @@
 		tau.engine.createWidgets(webclipsContainer);
 	}
 
+	function updateWebClipListPopup() {
+
+		var popupList = document.getElementById("popup-list");
+
+		// remove previous
+		popupList.childNodes.forEach(function (li) {
+			popupList.removeChild(li);
+		});
+
+		webClipList.forEach(function (webclip) {
+			var li = document.createElement("li"),
+				input = document.createElement("input"),
+				label = document.createElement("label"),
+				webClipName = webclip;
+
+			webClipName = webClipName.replace("webclip/", "");
```

Comment:
Ditto. Or, better idea, encapsulate this code into the function instead of repeat the lines.

Reply:
encapsualated in function